app/src/main/java/edu/wpi/first/shuffleboard/app/Main.java

```diff
@@ -10,11 +13,22 @@
  */
 @SuppressWarnings("PMD.UseUtilityClass") // Nope.
 public final class Main {
+  private static final Logger logger = Logger.getLogger(Main.class.getName());
+
   @SuppressWarnings("JavadocMethod")
   public static void main(String[] args) {
     // JavaFX 11+ uses GTK3 by default, and has problems on some display servers
     // This flag forces JavaFX to use GTK2
     System.setProperty("jdk.gtk.version", "2");
+
+    Thread
+        .setDefaultUncaughtExceptionHandler((thread, throwable) -> {
```

Comment:
I'd use a method reference instead of a lambda, just to make the log a bit more descriptive.

For reference, this is what logs would look like:
```
Jan 06, 2019 4:46:00 PM edu.wpi.first.shuffleboard.app.Main lambda$main$0
```